tools/dm/troubleshooting.md

```diff
@@ -32,7 +32,7 @@ In addition, the user of the upstream and downstream databases must have the cor
 
 ### Incompatible DDL statements
 
-When you encounter the following error, you need to manually handle it using dmctl (skipping the DDL statement or replacing the DDL statement with a specified DDL statement). For details, see [Skip or replace abnormal SQL statements](#skip-or-replace-abnormal-sql-statements).
+When you encounter the following error, you need to manually handle it using dmctl (skipping the DDL statement or replacing the DDL statement with a specified DDL statement). For details, see [Skip or replace abnormal SQL statements](/tools/dm/skip-replace-sqls.md).
```

Comment:
the CN doc is out-of-date, no *the following error* exists now.
so, may need to
1. change to something like `When you encounter an incompatible DDL in TiDB, you need to manually handle it using dmctl`
2. remove the code block below